hibernate-core/src/main/java/org/hibernate/type/spi/TypeConfiguration.java

```diff
@@ -301,6 +303,18 @@ public void addBasicTypeRegistrationContributions(List<BasicTypeRegistration> co
 		}
 	}
 
+	public void addCompositeUserTypeContributions(
```

Comment:
Please inline this into `MetadataBuildingProcess`.

Reply:
Done, thanks!